src/app/components/board-settings-dialog/board-settings-dialog.component.html

```diff
@@ -0,0 +1,16 @@
+<mat-toolbar class="settings-title" >Board Settings</mat-toolbar>
```

Comment:
perhaps "Board Details"?

Reply:
Probably just keep it as settings. Editor is kinda of subjective. Settings gets the point across and should we expand on this project later we can different features to it.